wrappers/rust/icicle-core/src/field.rs

```diff
@@ -82,6 +83,12 @@ impl<const NUM_LIMBS: usize, F: FieldConfig> FieldImpl for Field<NUM_LIMBS, F> {
         Self::from(limbs)
     }
 
+    fn from_hex(s: &str) -> Self {
+        let mut bytes = Vec::from_hex(&s[2..]).expect("Invalid hex string");
```

Comment:
Should we add a check that the what we are chopping is indeed a `0x`?

Reply:
good catch, will add. I just copy-pasted this function from HorizonLabs for test purposes